src/queries/CreateVehicleQuery.jsx

```diff
@@ -0,0 +1,16 @@
+import {gql} from "@apollo/client";
```

Comment:
this is not a query, but a mutation

so let's rename to `/src/mutations/CreateVehicleMutation`